tools/bumper/bumper.go

```diff
@@ -50,6 +50,10 @@ func main() {
 
 	var failedComponents []string
 	for componentName, component := range componentsConfig.Components {
+		if componentName == "kube-secondary-dns" {
+			logger.Printf("~~Skipping %s because of known problem with go-git~~", componentName)
```

Comment:
I would prefer that we fail and continue (so that the fail will be more visible and annoy us to fix it ASAP) but I don't insist on it for this PR.

Please add the link to the go-git issue

Reply:
Added https://github.com/go-git/go-git/issues/495